modules/ROOT/pages/business-groups.adoc

```diff
@@ -0,0 +1,111 @@
+= Business Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Business groups are self-contained resource groups that contain Anypoint Platform resources such as APIs and applications. Business groups provide a way to separate and control access to Anypoint Platform resources, as users have access only to the business groups in which they have a role. 
+
+Business groups reside within the master organization and are organized in a hierarchical tree where the top-level business group is the root. You can add up to 100 business groups per organization. Each business group you create has one direct parent and can have multiple children.
```

Comment:
100 BG limit is duplicated here and in the paragraph below. Can we remove it from here?